src/clarity-angular/utils/_variables.global.scss

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+$clr-default-borderradius: 0.125rem !default;
+$clr-default-borderwidth: 1px !default;
+
+// Hmmm, what to do with these?
+$clr-baseline: 24px !default;
+$clr-font-base-size: 14 !default; // Moved from original needs a better name
+$clr-rem-denominator: $clr-font-base-size !default; // Moved from original location needs better name
+//BaselineRem Variables
+$clr_baselineRem_0_1: baselineRem(0.1);
+$clr_baselineRem_0_125: baselineRem(0.125);
+$clr_baselineRem_0_2: baselineRem(0.2);
+$clr_baselineRem_0_25: baselineRem(0.25);
+$clr_baselineRem_0_3: baselineRem(0.3);
+$clr_baselineRem_0_375: baselineRem(0.375);
+$clr_baselineRem_0_5: baselineRem(0.5);
+$clr_baselineRem_0_72: baselineRem(0.72);
+$clr_baselineRem_0_75: baselineRem(0.75);
+$clr_baselineRem_1: baselineRem(1);
+$clr_baselineRem_1_25: baselineRem(1.25);
+$clr_baselineRem_1_5: baselineRem(1.5);
+$clr_baselineRem_1_75: baselineRem(1.75);
+$clr_baselineRem_2: baselineRem(2);
+$clr_baselineRem_2_5: baselineRem(2.5);
+$clr_baselineRem_3: baselineRem(3);
+$clr_baselineRem_3_5: baselineRem(3.5);
+$clr_baselineRem_4: baselineRem(4);
+$clr_baselineRem_5: baselineRem(5);
+$clr_baselineRem_6: baselineRem(6);
+
+//13. Outline Offset
+$clr-outline-offset: -5px;
+// App bg color
+$clr-global-app-background: $gray-lighter;
+// App selection color
+$clr-global-selection-color: #D9E4EA;
```

Comment:
We need to add...

`$clr-selection-color: $clr-global-selection-color;`

...here. Otherwise, the removal of `$clr-selection-color` will be a breaking change.

Please add a comment that `$clr-selection-color` is deprecated.

Reply:
I will add this.